config/routes.rb

```diff
@@ -146,10 +146,13 @@ def set_up_flipper
       patch :restore, on: :member
     end
     resources :partners do
+      resources :profiles, only: %i(edit update)
```

Comment:
```suggestion
      resource :profiles, except: %i(create)
```
I think with this you should be able to get the same desired routes without needing to specify member below.